src/NotificationHandler.php

```diff
@@ -62,7 +66,26 @@ public function handle($username, $password, $xml)
         }
 
         $event = $this->parse($xml);
+        $message = $event->getMessage();
+        $response = $message->generateResponse();
+        $status = new Status();
+
+        $status->setType('request')
+          ->setTime(date('d/m/Y', time()))
+          ->setDate(date('h:i:s', time()));
+        if ($event->getName() == ExceptionEvent::NAME) {
+            $status->setCode('-1')
+              ->setMessage('NOK');
+        } else {
+            $status->setCode('0')
+              ->setMessage('OK');
+        }
```

Comment:
This code is called from the SoapServer class actually, so nothing can be returned, we probably need a server class that extends or uses the soap server object and which can return status messages.